spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
+
+RSpec.describe 'User integrations', type: :feature do
+  it 'displays user profile in show page' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
+
+  it 'displays user username in show page' do
+    user = create(:user, username: 'acarnegie')
+    visit user_path(user)
+    expect(page).to have_content(user.username)
+  end
+  it 'displays total number of posts' do
+    user = create(:user)
+    create_list(:post, 3, user:)
+    visit user_path(user)
+    expect(page).to have_content("Posts Number: #{user.posts.count}")
+  end
+  it 'displays user bio' do
```

Comment:
- [ ] Good job for changing the tests that the previous reviewer suggested but I think it would be great if you fix the failing tests or you can write tests that fit your projects to pass well ⭐✅ so please fix your tests here 🙏🏻🙏🏻

![image](https://github.com/AlaaAlsalem/blogaapp/assets/58553711/71c05924-2aed-4fd6-b8c5-fcd221f3f6f5)